src/Angor/Client/Pages/Signatures.razor

```diff
@@ -292,17 +285,54 @@ else
                     var sigResJson = await encryption.DecryptNostrContentAsync(
                         nostrPrivateKeyHex, pendingSignature.investorNostrPubKey, pendingSignature.EncryptedMessage);
 
-                    pendingSignature.SignRecoveryRequest = serializer.Deserialize<SignRecoveryRequest>(sigResJson);
-
-                    if (pendingSignature.SignRecoveryRequest is null)
+                    if (!string.IsNullOrEmpty(sigResJson))
```

Comment:
no no Milad this is way too complicated code, we should role back to the code that was committed in my pr, there is no need for regex checks here

Reply:
I understand now, you are trying to support backwards compatibility.
I would argue we are on testnet there is no reason to support backwards compatibility on test data.
Projects that cant deserialize we just treat as spam messages.